### ISSUE TITLE
template literals and handling of missing semi-colons

### ISSUE THREAD

**Issue by futtta** [mrclay/jsmin-php#8]
When normal JS vars are on separate lines with missing semi-colons, they are left on separate lines, but when using template literals the line-break is removed resulting in broken JS.

JS in;
```javascript
// template literals without semicolons
var foo = `Hello`
foo = `world`

// template literals with semicolons
var foo = `Hello`;
foo = `world`;

// Simple strings without semicolons
var foo = 'Hello'
foo = 'world'
```

JS out:
```javascript
var foo=`Hello`foo=`world`var foo=`Hello`;foo=`world`;var foo='Hello'
foo='world';
```

**Comment by glensc** [comment]
Can you confirm this is still an issue?

as I see from 2.4.0 changelog this was addressed via #7:
- https://github.com/mrclay/jsmin-php/blob/2.4.1/HISTORY.txt#L6-L7

**Comment by futtta** [comment]
Just did a test with the current code on my localhost machine and yes, this is still an issue;
![image](https://user-images.githubusercontent.com/181297/160851144-7f6a3143-e553-4b4d-9754-150b305f30a3.png)

**Comment by glensc** [comment]
Please send a PR with a proposed fix. Also, include the exact version that you tested with.

**Comment by futtta** [comment]
afraid I don't have a proposed fix.

version tested was a fresh clone of the git repo, using master branch.

**Comment by glensc** [comment]
for posterity, include the actual version or commit hash.

**Comment by futtta** [comment]
git log says 012afaff2979fd903eee33a32650d8f9fdcb4c95 (i.e. you commit 3 days ago).

**Comment by PLUS-RobertEhrenleitner** [comment]
Addressed by [PR #17](https://github.com/mrclay/jsmin-php/pull/17)

**Comment by PLUS-RobertEhrenleitner** [comment]
I added a test case.

**Comment by glensc** [comment]
forgot to check PR uses [issue close patterns](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

closing manually

**Comment by glensc** [comment]
thanks. released as 2.4.3
